modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -9,6 +9,12 @@
 [role="_abstract"]
 GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to OpenShift Pipelines. Cluster administrators can configure a single GitHub App for all cluster users. For GitHub Apps to work with Pipelines as Code, ensure that the webhook of the GitHub App points to the Pipelines as Code event listener route (or ingress endpoint) that listens for GitHub events.
 
+[NOTE]
+====
+When importing an application using *Import from Git* and the Git repository has a '.tekton` directory, you can configure `pipelines-as-code` for your application. For more information see, xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-the-developer-perspective[Creating applications using the Developer perspective].
```

Comment:
Again, no xrefs in modules (yes, this makes me sad :-(  )

Reply:
I'll remove this! :(